src/core_plugins/kibana/public/visualize/wizard/_dialog.scss

```diff
@@ -0,0 +1,99 @@
+  background-repeat: no-repeat;
+  background-position: calc(100% + 1px) calc(100% + 1px);
+  background-size: 37%;
+}
+
+.visNewVisDialog__body {
+  display: flex;
+  padding: $euiSizeM $euiSizeL 0;
+}
+
+.visNewVisDialog__searchWrapper {
+  flex-shrink: 0;
+}
+
+.visNewVisDialog__typesWrapper {
+  max-width: $euiSizeXXL * 10;
+  padding-top: 2px; // Account for search field dropshadow
+  padding-bottom: $euiSize;
+  // Add overflow shadows via pseudo elements
+  position: relative;
+  &::before,
+  &::after {
+    content: "";
+    display: block;
+    position: absolute;
+    height: $euiSizeXXL;
+    left: 0;
+    right: 0;
+    pointer-events: none;
+  }
+
+  &::before {
+    top: -$euiSizeXXL + 2px; // Account for search field dropshadow
```

Comment:
nit: The `2px` value feels a little like a magic number, is it worth making it a variable? Or does EUI provide it as a variable?

Reply:
On second thought, if @cchaos already reviewed, I'm not really worried about it :)

Reply:
Actually she wrote that :-) It's coming in from her change PR.